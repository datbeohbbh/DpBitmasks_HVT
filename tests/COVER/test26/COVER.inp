705 79316 99881
65546 65661
67776 67854
43723 43804
61440 61478
64850 65077
10650 10843
23588 23613
43144 43300
89450 89477
73333 73502
25346 25416
68056 68198
5927 6129
84559 84570
83686 83768
85445 85667
13776 13885
43512 43629
14730 14968
6680 6734
14364 14588
14335 14364
72027 72275
13364 13467
3748 3802
5304 5333
90800 90940
73750 73884
31085 31262
93515 93573
41513 41525
67623 67639
99838 99967
74405 74548
17878 18119
94178 94379
88395 88531
65661 65812
95972 96049
95216 95278
1988 2036
4833 4850
44754 44917
70367 70589
68033 68056
1275 1448
98561 98766
9845 9889
60800 60847
76377 76575
18325 18462
90015 90264
10102 10274
14968 15162
13885 14005
67472 67601
97590 97608
2036 2043
8548 8732
68530 68634
29822 29854
1 117
42975 43144
98385 98477
99166 99320
96465 96622
97388 97497
70287 70364
24032 24051
3088 3260
70572 70817
92595 92749
92316 92371
8430 8433
16678 16788
13125 13171
67854 68033
28297 28376
85323 85445
83194 83324
23078 23162
8921 9100
70817 70853
70393 70435
64655 64704
90264 90370
13620 13863
85270 85284
42100 42105
8027 8139
42817 42975
30120 30331
85807 85853
95278 95385
72283 72488
25551 25768
69424 69468
4711 4833
63297 63452
99484 99594
6632 6780
75357 75541
13536 13620
31411 31659
22355 22448
62059 62130
44221 44374
25768 25995
71777 71922
14588 14730
2475 2507
4850 5032
3322 3332
91690 91905
89565 89801
70835 71067
5176 5400
3620 3748
90546 90714
83012 83100
91470 91690
23456 23588
65812 65960
62551 62588
9409 9507
27635 27827
30649 30702
4127 4226
68449 68503
60985 61177
71723 71851
73228 73318
91324 91470
74989 75023
1848 1912
23889 23929
85667 85755
96049 96234
3951 4183
88835 89046
77097 77311
73728 73750
62796 62895
64779 64826
8082 8251
72275 72350
28748 28943
16482 16673
8950 9078
8251 8420
7730 7852
13467 13542
17580 17640
23796 23889
3122 3156
94790 94885
10403 10563
5802 6033
89046 89076
2507 2565
41813 41968
407 637
25995 26060
75541 75753
16788 16947
1164 1267
96622 96856
4226 4417
65295 65392
85142 85270
31659 31751
4561 4711
696 911
83820 84067
76575 76604
15218 15352
70476 70528
71851 71856
87717 87792
9183 9250
12732 12881
61701 61772
45284 45513
85853 85903
87303 87464
61772 61830
65987 66235
10274 10403
71559 71666
78067 78251
41387 41513
18273 18325
7365 7423
93324 93400
13049 13177
9889 10102
85755 85807
13761 13776
14074 14106
87862 88067
70528 70572
91905 91936
3887 4127
92371 92595
11117 11310
17671 17878
67299 67472
22720 22789
12580 13125
30843 31085
62164 62167
41203 41357
25145 25346
1267 1326
45087 45284
93400 93515
99122 99166
9868 11807
66769 66844
3453 3660
349 476
14106 14111
75147 75357
71937 71997
16253 16482
14006 14127
90370 90546
91936 92152
63894 64031
188 293
5601 5725
71856 71859
8005 8082
42605 42778
295 349
93915 94010
86443 86666
85312 85323
98028 98227
4745 4965
94588 94634
3050 3122
15408 15590
26517 26745
15961 16140
75952 76086
3378 3584
7576 7691
66468 66571
14801 14856
92856 92865
66235 66458
12046 12273
72735 72876
24016 24032
29986 30120
30436 30649
32359 32501
8420 8430
8139 8265
71859 71937
23001 23024
30331 30436
8771 8921
1448 1689
44593 44754
23162 23374
7691 7730
44917 44961
74548 74597
83346 83448
11485 11584
93214 93324
28699 28748
75023 75147
72488 72679
41 188
8265 8327
15590 15729
29568 29786
8852 8950
8623 8771
32501 41203
73097 73333
69934 70183
18119 18273
76904 77097
89076 89202
69241 69424
9194 9317
62588 62796
2043 2220
87668 87717
42118 42235
26745 26902
13298 13367
70435 70476
82832 82894
64826 64850
44961 45087
9250 9443
84648 84818
14005 14074
71046 71287
62036 62059
62895 63072
83768 83820
15805 15961
96856 97017
13171 13364
5161 5229
90714 90723
73354 73457
69048 69241
63610 63645
11981 12121
11334 11485
76604 76685
6033 6105
42105 42118
7010 7170
94634 94790
66844 67092
10848 11025
24051 24232
61874 61956
10563 10650
14492 14631
77777 77892
69714 69733
98995 99043
83324 83346
92865 93091
72921 73030
73457 73466
65960 65987
9734 9845
62167 62336
8447 8548
27054 27103
29158 29328
5542 5559
2565 2654
83571 83686
84892 84900
63645 63832
94379 94477
63282 63297
93573 93666
24574 24805
70589 70835
44374 44593
76265 76377
18530 18614
74081 86514
28536 28699
1921 2167
23024 23078
72679 72921
7170 7307
64704 64779
8327 8447
6105 6307
84339 84559
32175 32311
87082 87143
88759 88835
26902 26935
22848 23001
95729 95972
89801 89942
9507 9619
28376 28536
97017 97182
4417 4517
14127 14305
12273 12419
73502 73673
63452 63521
71402 71561
30702 30832
63832 63894
11819 12046
72876 73097
16673 16678
1105 1131
44029 44221
96234 96413
73829 73975
98766 98995
5333 5542
7423 7453
3584 3620
87861 87967
2842 3050
9501 9734
62035 62036
27259 27504
15546 15760
5032 5176
63072 63239
74597 74776
11807 11981
86005 86227
4567 4745
13863 14006
73797 73829
1864 1988
86849 87082
5748 5849
13542 13761
87530 87668
87711 87861
61956 62035
10843 10848
1689 1757
77311 77459
89477 89565
7307 7448
65424 65546
77657 77777
911 951
2784 2842
3660 3887
16140 16253
7632 8027
32064 32175
31872 32064
1912 1921
7029 7274
86227 86316
89942 90015
71067 71252
13367 13536
22448 22490
63521 63610
88531 88759
77459 77657
71561 71595
15729 15805
43984 44029
3156 3284
3284 3378
71997 72150
72350 72490
69733 69838
66571 66769
17246 17491
71252 71402
6555 6680
4517 4561
42778 42817
87792 87862
97974 98022
9100 9183
25416 25551
45513 45530
93666 93915
1816 1864
92152 92316
14111 14124
87231 87381
67639 67663
97827 97974
2847 3088
94946 95184
87464 87711
98022 98028
3294 3322
73673 73728
61830 61874
1757 1762
41357 41387
12430 12574
71595 71723
15760 15968
62130 62164
2424 2569
7448 7632
73975 74066
68503 68530
22789 22848
92749 92856
7274 7365
262 295
7453 7576
17491 17580
24232 24454
84130 84139
87381 87530
6874 7010
70141 70287
637 696
88067 88288
68671 68824
99594 99838
7852 8005
22490 22720
84900 84988
73030 73228
27827 27989
71287 71386
88161 88395
15968 60800
2220 2424
87143 87303
60949 60985
6492 6533
86841 87006
709 903
97182 97388
86821 86841
70194 70367
69002 69048
14856 15015
68634 68671
73737 73797
70853 71046
76685 76904
17188 17246
1745 1848
89202 89450
14305 14492
27103 27259
6780 6874
63239 63256
132 262
29854 29986
86316 86443
83100 83194
42359 42413
12419 12580
6174 6410
14124 14335
43804 43984
2569 2784
24454 24574
84570 84642
84642 84648
14631 14801
6734 6788
66458 66468
67666 67776
70080 70141
76086 76265
5400 5601
4183 4403
95385 95558
95184 95216
2348 2475
71666 71777
26935 27054
84067 84130
91079 91324
30832 30843
74066 74171
69468 69714
78400 82832
31262 31411
86752 86821
71422 71559
70183 70194
99967 100000
41666 41813
1526 1634
5849 5927
4403 4567
69920 69934
2167 2348
61177 61331
68824 68855
26060 26307
67601 67623
12574 12732
78251 78400
11310 11334
85115 85142
3260 3294
23374 23456
68398 68449
90723 90800
67092 67299
9317 9409
85903 86005
61478 61515
68198 68398
60847 60949
31751 31872
12881 13049
70364 70393
12349 12430
99043 99122
5229 5304
903 1105
64305 64550
41525 41666
63256 63282
86666 86849
61331 61440
65077 65295
26307 26517
12121 12349
4965 5161
77981 78021
82894 83012
95715 95729
15162 15408
11025 11117
87967 88161
9078 9194
75753 75952
1131 1275
98227 98385
61515 61701
93091 93214
96413 96465
64058 64305
99320 99484
8732 8852
8433 8623
117 132
42413 42605
1762 1816
74776 74989
84818 84892
65392 65424
72490 72735
3802 3907
28946 29158
84988 85115
5725 5802
97608 97827
67663 67666
94477 94588
9619 9868
97523 97590
18462 18530
73884 74021
97497 97523
6788 7029
68855 69002
77892 77981
43629 43723
27504 27635
13177 13298
6533 6632
5559 5748
1 41
73466 73537
9443 9501
24805 24908
6410 6492
29328 29568
28943 28946
15015 15218
41968 42100
3907 3951
95558 95715
94885 94946
45530 45765
24908 25145
32311 32359
90940 91079
1634 1745
293 407
6307 6555
73537 73737
42235 42359
78021 78067
43300 43512
86514 86752
62336 62551
3332 3453
18614 22355
23929 24016
15352 15546
11584 11819
2654 2847
87006 87231
28115 28297
1326 1526
64550 64655
45765 69854
23613 23796
16947 17188
83448 83571
74171 74405
27989 28115
98477 98561
94010 94178
69854 70080
71386 71422
72150 72283
84139 84339
64031 64058
74021 74081
71922 72027
29786 29822
17640 17671
73318 73354
6129 6174
69838 69920
476 709
951 1164
85284 85312
