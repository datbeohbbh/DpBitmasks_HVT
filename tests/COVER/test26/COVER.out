100
9
27
30
33
36
37
39
40
47
52
57
64
65
66
67
72
73
86
94
100
114
115
119
121
133
138
140
156
159
167
194
202
204
216
217
227
228
236
239
241
251
273
288
309
312
322
331
333
347
349
355
367
370
371
374
388
390
394
409
413
418
432
437
454
458
460
462
464
466
467
468
483
490
496
503
511
522
548
549
554
581
592
598
606
609
613
615
616
618
632
634
636
639
658
659
663
671
679
689
690
